src/controllers/v1/role-extension.js

```diff
@@ -0,0 +1,40 @@
+const common = require('@constants/common')
+const roleExtensionService = require('@services/role-extension')
+
+module.exports = class Reports {
+	async create(req) {
+		try {
+			const createReport = await roleExtensionService.createRoleExtension(req.body)
+			return createReport
+		} catch (error) {
+			return error
+		}
+	}
+
+	async getRoleExtension(req) {
+		try {
+			const getReportById = await roleExtensionService.getRoleExtension(req.query.title)
```

Comment:
@sumanvpacewisdom use the proper naming convention,  define variable name as roleExtensionDetails